Makefile

```diff
@@ -23,13 +23,12 @@ REGISTRY_LOCATION=us-central1
 
 .PHONY: setup
 setup:
-	curl -L -o ${OUTPUT_DIR}/ocb --create-dirs https://github.com/open-telemetry/opentelemetry-collector/releases/download/v${OTEL_VERSION}/ocb_${OTEL_VERSION}_linux_amd64
-	chmod +x ${OUTPUT_DIR}/ocb
-
+	go install go.opentelemetry.io/collector/cmd/builder@v${OTEL_VERSION}
```

Comment:
I opted not to change it here, but maybe the `v` should be part of the `OTEL_VERSION` variable? You could even rename the variable to be more specific like `OTEL_VERSION_TAG`. Up to you, probably out of scope for this PR either way.